sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md

```diff
@@ -159,13 +159,12 @@ input-file:
   -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/2fd7dcd89afa70ff5ba7be88bee987da62099a28/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json
 ```
 
-## Make generated models internal by default
+## Move generated models to the DocumentAnalysis namespace
 ``` yaml
 directive:
   from: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/2fd7dcd89afa70ff5ba7be88bee987da62099a28/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json
   where: $.definitions.*
   transform: >
-    $["x-accessibility"] = "internal";
```

Comment:
We were forcibly making generated models internal by default. This is a pattern we inherited from the old `FormRecognizerClient` because the convenience layer we used at the time was reasonably different from the generated layer. For that reason, most generated models were made internal and had a public wrapper type manually written around them.

With the new `DocumentAnalysisClient` we represent models very similarly to what we get from the service. This autorest transform doesn't help much given that most models will be made public anyway, so we're removing it.

Reply:
Thanks for cleaning this. Any reason why we are using so many transformations? Should we not consider requesting the service team to update the swagger, if possible?
Also, is this only needed by the  .NET language or others as well. if everyone, there is further benefit in making the change up the stream.

Reply:
- Most of the transforms we had for v2.x (old service) were caused by minor inconsistencies between the swagger and the service response. In particular, they used to return some fields as `null` even though this was not allowed without setting the `x-nullable` property to `true` in the swagger specification. We've brought up this matter a couple of times but it has never been fixed on their side, so we had no choice but to keep the transforms. When running in debug mode, our generated code does an assertion and throws if the service sends back an unexpected `null`, which may not be true for other languages.
- For v3.x (new service), most transformations are used to avoid conflicts between the new and the old models since the service kept the same name for some of them. Also, we're moving models to the new `DocumentAnalysis` namespace.